apps/web-giddh/src/app/shared/header/components/group-update/group-update.component.ts

```diff
@@ -514,11 +520,20 @@ export class GroupUpdateComponent implements OnInit, OnDestroy, AfterViewInit {
         this.destroyed$.complete();
     }
 
-
-
-
-
-
-
+    /**
+    * To get sub group belongs to mentioned group (currentliabilities , currentassets)
+    *
+    * @param {*} item
+    * @memberof GroupUpdateComponent
+    */
+    public isDebtorCreditorGroup(itemUniqueName: any): boolean {
+        let isTaxableGroup: boolean = false;
+        this.groupWithParentLessDetailsList.forEach(element => {
```

Comment:
Use find method instead of forEach as it will stop the iteration the moment it finds the account

Reply:
this is tree traversal so need to go in depth